src/Shared/Components/FilterChips/types.ts

```diff
@@ -37,10 +37,15 @@ export interface FilterChipProps {
      * If passed, the label will be formatted accordingly
      */
     getFormattedValue?: (filterKey: string, filterValue: unknown) => ReactNode
+    /**
+     * If true, would hide the label
+     * @default false
+     */
+    shouldHideLabel?: boolean
```

Comment:
suggestion: we can improve the typing here

Reply:
This is for internal component which is not exported and we are sending it as variable and could have made it mandatory as well but for parent which is extending this is consuming it as partial, which is result of initial bad typing since we should derive types from parent to child not other way around.